src/views/login.vue

```diff
@@ -0,0 +1,141 @@
+<template>
+  <div class="login__container">
+    <form @submit.prevent="loginUser">
+      <div class="container">
+        <label for="email"><b>Email</b></label>
+        <input
+          v-model="email"
+          type="text"
+          placeholder="Enter Email"
+          name="email"
+          id="email"
+          required
+        />
+        <label for="psw"><b>Password</b></label>
+        <input
+          v-model="password"
+          type="password"
+          placeholder="Enter Password"
+          name="psw"
+          required
+        />
+        <button type="submit">Login</button>
+        <label>
+          <input type="checkbox" checked="checked" name="remember" /> Remember
+          me
+        </label>
+      </div>
+      <div class="container" style="background-color: #f1f1f1">
+        <button type="button" class="cancelbtn">Cancel</button>
+        <router-link to="/register"
+          ><span class="psw"
+            >Not Registred?<a href="#">Register</a></span
+          ></router-link
+        >
+      </div>
+    </form>
+  </div>
+</template>
+
+<script>
+import axios from "axios";
+export default {
+  data() {
+    return {
+      email: "",
+      password: "",
+    };
+  },
+  methods: {
+    loginUser() {
+      axios
+        .post(
+          "https://documentation-vue.projects.codennection.pl/api/auth/login",
+          {
+            email: this.email,
+            password: this.password,
+          }
+        )
+        .then((response) => {
+          this.$store.dispatch(
+            "Auth/login",
+            response.data.data.email,
+            response.data.data.password
```

Comment:
Nie zwracaj aż 3 parametrów. Funkcja dispatch powinna przyjmować tylko dwa: nazwę wywoływanej akcji i payload, który jest obiektem. https://vuex.vuejs.org/guide/actions.html#dispatching-actions

Reply:
Napisałeś że payload, który jest obiektem ? Nie wiem o co się rozchodzi

Reply:
Czytałeś wgl dokumentację, którą wysłałem? Payload to jest obiekt, który podajesz jako drugi argument funkcji dispatch i odbierasz go w akcji jako drugi parametr. Powinno być:
 this.$store.dispatch(
            "Auth/login",
            {
            token: response.data.data.token,
            user: response.data.data.user,
            }
)